valhalla/baldr/rapidjson_utils.h

```diff
@@ -12,6 +12,7 @@
 #include <boost/optional.hpp>
 
 // rapidjson loves to assert and crash programs, its more useful to throw and catch
+#define RAPIDJSON_ASSERT_THROWS
```

Comment:
by doing this with a newer version of rapidjson, you can remove the warning you get when their internal functions offer nothrow gaurantees but you've overridden the assertions with throws. there should be some docs on the rapidjson side about how to effectively use this. i think this is a very common use-case tbh. more info here: https://github.com/Tencent/rapidjson/issues/1280